src/Routers/SettingsRouter.js

```diff
@@ -1,26 +1,30 @@
 import PromiseRouter from '../PromiseRouter';
 import * as middleware from "../middlewares";
 import { logger, configureLogger } from '../logger';
+import SettingsManager from '../SettingsManager';
 import winston from 'winston';
 
 export class SettingsRouter extends PromiseRouter {
   mountRoutes() {
     this.route('GET', '/settings', middleware.promiseEnforceMasterKeyAccess, (req) => {
       return Promise.resolve({
-        response: {
-          logLevel: winston.level
-        }
-      })
+        response: SettingsManager(req.config.applicationId).getUnlocked()
```

Comment:
I'll try to clarify this a bit in another commit.  In essense, this code does return most valid settings.

The files, push, logger, etc controller objects that are cached are the only ones that should be excluded.